src/hooks/useTokenDisplayData.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import { type EdgeCurrencyConfig, type EdgeCurrencyWallet } from 'edge-core-js'
+import { type EdgePluginMap } from 'edge-core-js/types'
+
+import { useSelector } from '../types/reactRedux'
+import { getAllTokens } from '../util/CurrencyInfoHelpers'
+import { fixFiatCurrencyCode, getDenomFromIsoCode, getYesterdayDateRoundDownHour, zeroString } from '../util/utils'
+
+/**
+ * Returns data from tokens relevant for display
+ * */
+
+export const useTokenDisplayData = (props: { currencyConfigMap: EdgePluginMap<EdgeCurrencyConfig>, tokenId?: string, wallet: EdgeCurrencyWallet }) => {
```

Comment:
You do not need `currencyConfigMap`. You can just access `EdgeWallet.currencyConfig` to get to the right one directly.